### PR TITLE
Use github_api_token secret for interact server

### PR DESCRIPTION
**Pull request by SamLau95**
This is a token on my Github account (I filled in secrets.vault.yml on ds8 and
data8). This will allow the interact server to pull from private repos.

Wanted to use a PR to run this by you first to confirm that I can pull
from `master` on ds8 and `azure` on data8 (?) after this is merged into
both branches.

@ryanlovett

### CODE REVIEW

**Comment by SamLau95** [comment]
I also wonder whether it's possible to pass in the base URL for ds8/data8 as an env variable into the interact server so I won't have to have two different branches on that repo.

File: roles/interact/tasks/main.yml [hunk 1]
```diff
@@ -49,7 +52,11 @@
     - rebuild-interact
 
 - name: start interact
-  command: env JPY_API_TOKEN={{ jpy_api_token.stdout }} daemon -n interact -o /var/log/interact.log --chdir=/srv -- python3 /srv/interact/run.py --production
+  command: >
+    env JPY_API_TOKEN={{ jpy_api_token.stdout }}
+    env GITHUB_API_TOKEN={{ github_api_token }}
```
Comment (ryanlovett) [inline roles/interact/tasks/main.yml]: A spurious env here, but looks fine otherwise.

**Comment by ryanlovett** [comment]
And yes, you can pass in the base URL as an env var too.

**Comment by SamLau95** [comment]
Cool, can you look over the way I pass in the base url there? Not sure if it was already defined somewhere else so I defined a `base_url` variables in `secrets`.

**Comment by SamLau95** [comment]
Feel free to merge if you've got no other comments.

**Pull request merged**